### DISCUSSION TITLE
CSS 19 de Octubre

### DISCUSSION THREAD

**Discussion by Kariid26** [Launch-X-Latam/MisionFrontEnd#246] [Q&A]
Buenos días compañeros, alguien sabe cómo puedo accesar a la grabación de la clase de CSS del día 19 de Octubre? Busque la grabación en stream que es donde han subido las anteriores pero creo que no aún no la suben

**Comment by iam-leo** [comment]
Buenos días explorer! Puedes ingresar en teams y dentro de los canales ve al Módulo 3 - CSS y ahí buscas el link de la transmisión.<br>
<div align="center">
    <img src="https://user-images.githubusercontent.com/79548542/197350440-6f358aa1-a9d3-44d5-bb8a-1de9381fc933.png" width="700px">
</div>
<br>
o sino aquí te dejo el link para que lo veas desde el canal del MC en ms streams <a href="https://web.microsoftstream.com/video/9a9b6af7-aca5-4daf-890e-16037fb32aba?list=user&userId=96eeb66f-b984-4083-8ea6-74c34a07a34a"> Ver transmisión </a>

**Reply by Kariid26** [reply]
Muchas gracias explorer! Ya intente ver la grabación por Teams y por ms Stream pero me aparece un error, no se si sea error de mi dispositivo. ¿A ti si te deja reproducirlo?

**Reply by iam-leo** [reply]
A mi me está funcionando bien tanto en teams como ms stream :confused:. Puedes probar cerrar sesión, borrar caché y volver a ingresar a ver si el error persiste.

**Reply by Kariid26** [reply]
Ahora si ya me dejo reproducirlo🤗🙌 Muchas gracias por la ayuda!!

**Reply by iam-leo** [reply]
Genial! De nada compa :wink: :raised_hands:
